src/components/VJsoneditor/VJsoneditor.js

```diff
@@ -21,7 +21,6 @@ export default {
       },
     },
     value: {
-      type: Object,
```

Comment:
Could you add type: [Object, Array] so we are more restrictive regarding JSON data?

Reply:
Actually, strings, numbers, booleans and nulls are valid JSONs too, and they all can be inserted thru the JSON Editor (for example, using the Code mode, or by changing the type of the root object in the Tree mode).